endpoints/getting-started/clients/src/main/java/com/example/app/JWTClient.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.app;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+
+import java.io.FileInputStream;
+import java.security.interfaces.RSAPrivateKey;
+
+
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTCreator;
+
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+
+import java.net.URL;
+import java.net.HttpURLConnection;
+import java.net.ProtocolException;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+/**
+ * JWTClient shows how a client can authenticate with a Cloud Endpoints service
+ */
+public class JWTClient {
+
+// [START endpoints_generate_jwt_sa]
+  /**
+   * Generates a signed JSON Web Token using a Google API Service Account
+   * utilizes com.auth0.jwt.
+   */
+  public static String generateJWT(final String saKeyfile, final String saEmail,
+    final String audience, final int expiryLength)
+    throws FileNotFoundException, IOException {
+
+    Date now = new Date();
+    Date expTime = new Date(System.currentTimeMillis() + TimeUnit.SECONDS.toMillis(expiryLength));
+
+    // build jwt
+    JWTCreator.Builder token = JWT.create()
+        .withIssuedAt(now)
+        // expires after 'expirary_length' seconds.
```

Comment:
Please use capitalization and be consistent with punctuation (either use it for all comments or no comments) 

(same for the rest of the comments)